app/concepts/tickets/repository.rb

```diff
@@ -3,5 +3,9 @@ class Repository < ::BaseRepository
     def initialize(adapter: Ticket)
       super(adapter: adapter)
     end
+
+    def find(id, reservation_id)
+      adapter.where(id: id, reservation_id: reservation_id).first!
```

Comment:
you could also use `find_by!` 
https://apidock.com/rails/v4.0.2/ActiveRecord/FinderMethods/find_by%21

Reply:
You are right, thank you! :)